internal/rest/avi_evh_obj.go

```diff
@@ -215,12 +223,20 @@ func (rest *RestOperations) EvhNodeCU(sni_node *nodes.AviEvhVsNode, vs_cache_obj
 
 		// Not found - it should be a POST call.
 		restOp := rest.AviVsBuildForEvh(sni_node, utils.RestPost, nil, key)
-		rest_ops = append(rest_ops, restOp...)
+		if restOp != nil {
+			rest_ops = append(rest_ops, restOp...)
+		}
 	}
 	return cache_sni_nodes, rest_ops
 }
 
 func (rest *RestOperations) AviVsBuildForEvh(vs_meta *nodes.AviEvhVsNode, rest_method utils.RestMethod, cache_obj *avicache.AviVsCache, key string) []*utils.RestOp {
+
+	if len(vs_meta.Name) > lib.AVI_OBJ_NAME_MAX_LENGTH {
```

Comment:
same here, we can use the `PrintObjectNameLengthWarn` function here, and in other build functions too.